sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -430,37 +428,18 @@ export type ReceiptItemField = {
 } & CommonFieldValue;
 
 // @public
-export type ReceiptPollerLike = PollerLike<PollOperationState<RecognizeReceiptResultResponse>, RecognizeReceiptResultResponse>;
+export type ReceiptPollerLike = PollerLike<PollOperationState<RecognizedReceiptArray>, RecognizedReceiptArray>;
 
 // @public (undocumented)
-export type ReceiptWithLocale = {
-    locale: "US";
-} & USReceipt;
-
-// @public
-export type RecognizeContentOperationResult = Partial<RecognizedContent> & {
-    status: OperationStatus;
-    createdOn: Date;
-    lastModified: Date;
-};
+export interface ReceiptWithLocale {
```

Comment:
what does this type represent? In .NET and Python we call it `RecognizedReceipt`. Note that Python has the properties of `RecognizedForm` flattened her. See https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py#L182

FYI @kristapratico in case I am reading Python wrong

Reply:
agree, this would fit most closely with what other languages call `RecognizedReceipt`

Reply:
I have `RecognizedReceipt` too.  `ReceiptWithLocale` is more for intermediate use before we cast it to `RecognizedReceipt` with more specific locale like "US", "UK", etc.

Reply:
could you show me how the user will use it? not sure I understand the scenario

Reply:
The user does not use it.  It's mostly internal like a base class.  However I may be able to remove it.

Reply:
Removed in https://github.com/Azure/azure-sdk-for-js/pull/8933/commits/9ada2265a797c405e4efebe72f37ef8c6241fa4f.

It was holding the common properties for USReceipt/UKReceipt/etc.  However since there's only two properties I can duplicate them in each type.

Reply:
ohh thanks for the clarification. 
My apologies. Still learning the JS repo project structure

Reply:
I think I still want the common fields back to enforce that all RecognizedReceipts have them.  Using mixins this time https://github.com/Azure/azure-sdk-for-js/pull/8933/commits/5149354273199f6b8b3dca765e3484718e77ada9